Lib/test/test_range.py

```diff
@@ -519,13 +519,14 @@ def test_range_iterators(self):
             iter1 = range(start, end, step)
             iter2 = pyrange(start, end, step)
             test_id = "range({}, {}, {})".format(start, end, step)
-            # check first 100 entries
-            self.assert_iterators_equal(iter1, iter2, test_id, limit=100)
+            # check first 10 entries
+            self.assert_iterators_equal(iter1, iter2, test_id, limit=10)
```

Comment:
Why reduce from 100 to 10?

Reply:
I made it 4x slower otherwise by doubling the number of `M` values, and it started to feel a bit sluggish, so I wanted to offset that.